src/main/java/com/cdancy/bitbucket/rest/domain/build/Status.java

```diff
@@ -31,21 +32,23 @@
 
     public abstract long dateAdded();
 
+    @Nullable
     public abstract String description();
 
     public abstract String key();
 
+    @Nullable
     public abstract String name();
 
     public abstract StatusState state();
 
     public abstract String url();
 
     @SerializedNames({"dateAdded", "description", "key", "name", "state", "url"})
-    public static Status create(final long dateAdded, 
-            final String description, 
-            final String key, 
-            final String name, 
+    public static Status create(final long dateAdded,
+            @Nullable final String description,
+            final String key,
+            @Nullable final String name,
```

Comment:
You can remove these 2 @Nullable annotations as it's only required on the variable itself for AutoValue to correctly handle things.